src/app/embed/adapters/feed.adapter.ts

```diff
@@ -132,7 +132,7 @@ export class FeedAdapter implements DataAdapter {
   processEpisode(item: Element, props: AdapterProperties = {}): AdapterProperties {
     props.title = this.getTagText(item, 'title');
     props.audioUrl = this.getTagTextNS(item, FEEDBURNER_NAMESPACE, 'origEnclosureLink')
-                  || this.getTagAttribute(item, 'enclosure', 'url');
+                  || this.getTagAttribute(item, 'enclosure', 'url') || null;
```

Comment:
This handles the case when the player is set on an episode in the feed that has no audio.
At least now renders the rest of the player, rather then all gray.